src/hooks/use-add-site.ts

```diff
@@ -68,7 +68,7 @@ export function useAddSite() {
 						} );
 						clearImportState( newSite.id );
 					}
-					await startServer( newSite.id );
```

Comment:
It's redundant, since we are starting server as in the previous step above https://github.com/Automattic/studio/blob/trunk/src/hooks/use-import-export.tsx#L134